### PR TITLE
fix typo "resouces"

### LINKED ISSUE #2333: Typo: "Resouces" instead of "Resources"
> If you are reporting an issue with our website, please specify the URL you were looking at, which browser and OS you are using, and if possible, include a screenshot showing the problem.

It is in the navigation menu and thus viewable across the site, but I first noticed when navigating from: https://obofoundry.org/principles/fp-000-summary.html
**Browser:** Chrome
**OS:** Windows 10

**Issue:** Navigation menu item says "Resouces" instead (I assume) "Resou**r**ces"
**Navigation:** _hamburger menu_ > About > Resouces

image

### PR DESCRIPTION
**Pull request by nlharris**
fixes #2333

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
                     <ul class="dropdown-menu" aria-labelledby="navbarAboutDropdown">
                         <li><a class="dropdown-item" href="/about-OBO-Foundry.html">About</a></li>
                         <li><a class="dropdown-item" href="/docs/COC.html">Code of Conduct</a></li>
-                        <li><a class="dropdown-item" href="/resources">Resouces</a></li>
+                        <li><a class="dropdown-item" href="/resources">Resourfces</a></li>
```
Comment (cthoyt) [inline _includes/header.html:22]:
```suggestion
                        <li><a class="dropdown-item" href="/resources">Resources</a></li>
```

Comment (nlharris) [reply _includes/header.html:22]: Omggg i can't believe i introduced ANOTHER typo. must stop doing PRs while asleep. 🤭

**Review by matentzn — APPROVED** [review]

**Comment by nlharris** [comment]
thanks for noticing the additional typo, @cthoyt, and for approving, @matentzn. can someone merge this change?

**Review by cmrn-rhi — APPROVED** [review]

**Comment by nlharris** [comment]
please someone merge this and put this embarrassing typo (and even more embarrassing botched attempt to fix it) out of its misery!

**Comment by matentzn** [comment]
@erik-whiting your domain!

**Pull request merged**